poc/common.sage

```diff
@@ -48,7 +48,10 @@ def zeros(length):
 # Return the requested number of random bytes.
 def gen_rand(length):
     if TEST_VECTOR:
-        return bytes([0x01] * length)
+        out = []
+        for i in range(length):
+            out.append(i % length)
+        return bytes(out)
```

Comment:
`i % length` seems strange, as it will never wrap around
```suggestion
        return bytes(int(i & 0xff) for i in range(length))
```

Reply:
Uhhh yup, this was supposed to be `256`. Good catch.